exercises/julia-1-a/README.md

```diff
@@ -0,0 +1,98 @@
+## Instructions
+
+Implement your own [tic-tac-toe](https://en.wikipedia.org/wiki/Tic-tac-toe) game.
+(_You may also know the game as "noughts and crosses" or "Xs and Os"._)
+
+The games is played on a 3x3 grid.
+Players take turns to place `X`s and `O`s on the grid.
+The game ends when one player has won by placing three of marks in a row, column, or along a diagonal of the grid, or when the entire grid is filled up.
+
+Without loss of generality we will assume that `X` starts.
+
+Create a function `gamestate` that takes the game board represented a string as its sole argument and returns a Symbol describing the state of the game.
+The board will be given as a 3-line string containing `X`s, `O`s and ` `s:
+
+```julia
+board = "X  \n XO\nOX "
+```
+
+represents the following board:
+
+```
+   |   |   
+ X |   |   
+___|___|___
+   |   |   
+   | X | O 
+___|___|___
+   |   |
+ O | X |  
+   |   |
+```
+
+There are 3 potential game states:
+
+- The game is **ongoing**.
+- The game ended in a **draw**.
+- The game ended in a **win**.
+
+If the given board is invalid, throw an appropriate error.
```

Comment:
Do you think this is clear enough or should I explain what kinds of boards would be invalid?

The tests check for taking turns out of order or boards that have multiple rows/cols/diags with 3 in a row.

Reply:
It might be good to explicitly list the invalid states, as that means less thinking by the student and more coding (which is what we want).

Reply:
Done in 1414171